geoportailv3/static/js/mymaps/mymapsdirective.js

```diff
@@ -45,6 +48,12 @@ app.module.directive('appMymaps', app.mymapsDirective);
 app.MymapsDirectiveController = function($scope, $compile, gettext) {
 
   /**
+   * @type {ol.interaction.Select}
+   *  @export
```

Comment:
As far as I understand, if I don't export it, I won't be able to use the dot notation.